webrender/src/texture_cache.rs

```diff
@@ -779,6 +779,13 @@ impl TextureCache {
         })
     }
 
+    pub fn add_raw_remove(&mut self, id: TextureId) {
```

Comment:
yeah, it does follow the convention. Perhaps, the convention should be revised? as a separate PR